charts/aws-adapter/values.yaml

```diff
@@ -30,7 +30,7 @@ image:
   repository: ghcr.io/nirmata/aws-adapter
   pullPolicy: Always
   # Overrides the image tag whose default is the chart appVersion.
-  tag: v0.1.0
+  tag: latest
```

Comment:
we should not use the latest as tag , instead the version should be v0.1.1 ?

Reply:
Fixed 🫡